docs/README.md

```diff
@@ -2,35 +2,49 @@
- 
-## Installation and Use:
+Also, while there are many plugins out there that "borrow" the name and even icon of NoChatReports, please be aware that **I have not authored any of them** and cannot say how well they do their job.
 
-This mod supports both Forge and Fabric, just download the version for respective modloader. It can be installed on either client, server, or both sides, and will function differently depending on which sides it is present at:
+## Installation and Usage:
 
-**1. Only Client:** Client will refuse to send account's public key to the server, and signatures will be stripped from the messages that you send. This way it won't be useful to try and report your messages, as there will be no proof they were actually sent from your account. Server will relay them unless `enforce-secure-profile` option is set to true in server options (which it is by default since 1.19.1), in which case you will not be able to join, unless you agree to send signed messages (the mod will supply warning screen);
+NoChatReports supports both Fabric and Forge, just download the version for your respective Mod-Loader and game version. It can be installed on either the client, server, or on both sides, and will function differently depending on which sides it is present on:
 
-**2. Only Server:** Clients will still attach signatures when sending messages to the server, but server will strip them before relaying them to other players. This way chat reporting will not work for any players that join. You can enable player to system message conversion in config, to prevent players without the mod from seeing them as "Not Secure"
+**1. Only Client:** The Client will refuse to send the Account's public key to the server, and signatures will be stripped from the messages that you send. This way it won't be useful to try and report your messages, as there will be no proof they were actually sent from your account. Server will relay them unless the `enforce-secure-profile` option is set to `true` in the `server.properties` file (which it is by default since 1.19.1), in which case you will not be able to join, unless you agree to send signed messages (NoChatReports will supply a warning screen)
 
-**3. Both Client and Server:** Signature will be stripped on client side before sending messages to server, server will make no attempt to verify message signatures. Chat reporting and "Only Show Secure Chat" will not function, and players will be notified that those features are disabled by the mod when trying to use them.
+**2. Only Server:** Clients will still attach signatures when sending messages to the server, but the server will strip them before relaying them to other players. This way chat reporting will not work for any players that join. You can enable the conversion of player to system messages in the config, to prevent players without the mod from seeing them as "Not Secure"
```

Comment:
Notably, since 1.20.2 if I recall correctly (check changelogs), NCR will not hide the client side unsigned messages popup via the server - only when installed on client.

Reply:
So, should I add another sentence along the lines of "On 1.20.2 and above, NoChatReports will not prevent clients from being informed that chat messages are unsigned"?

I hope that I understood this correctly.

Reply:
Yeah, how about this:
```suggestion
**2. Only Server:** Clients will still attach signatures when sending messages to the server, but the server will strip them before relaying them to other players. This way chat reporting will not work for any players that join. You can enable the conversion of player to system messages in the config, to prevent players without the mod from seeing them as "Not Secure". As of 1.20.2, the unsigned messages warning cannot be disabled servers-side, only with the client mod.
```

Reply:
Does this also work?
> As of 1.20.2 the server cannot disable the "unsigned messages" warning on the client. It can only be disabled if the client has NoChatReports (or a similar mod) installed.

Reply:
I have already added that (would've changed it if you'd said that it's not acceptable/the other one is better)

Is there anything else you want me to change in this PR?